presentation/common/src/presentation-common/RpcRequestsHandler.ts

```diff
@@ -213,4 +202,29 @@ export class RpcRequestsHandler implements IDisposable {
     return this.request<KeySetJSON, SelectionScopeRequestOptions<IModelRpcProps>>(
       this.rpcClient.computeSelection.bind(this.rpcClient), options, ids, scopeId);
   }
+
+  private static removeUnneededPropertiesFromRuleset(ruleset: any) {
```

Comment:
I'd move the function out of `RpcRequestsHandler` class and rename it to `cleanupRuleset(ruleset: Ruleset)`.